### PR TITLE
Feature/start integration tests

### LINKED ISSUE #20: Configure CI
- [x] Configure circle-ci to run on each builds
- [x] Start to test implemented code

### PR DESCRIPTION
**Pull request by cquennouelle**
First test of the API on the business.

### CODE REVIEW

File: integtests/IntegTests/HBSeriesDto.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace IntegTests
+{
+    /// <summary>
+    /// Class representing a Series
+    /// </summary>
+    class HBSeriesDto
```
Comment (bat3) [inline integtests/IntegTests/HBSeriesDto.cs]: Why create this class ? You cannot use the one from metadatadatbase ?

File: integtests/IntegTests/SeriesApiTest.cs [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+namespace IntegTests
+{
+    public class SeriesApiTest : IDisposable
+    {
+        #region Integration tests
+        [Fact]
+        public void GetSeries()
+        {
+            var response = client.Get(getSeriesRequest);
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
```
Comment (bat3) [inline integtests/IntegTests/SeriesApiTest.cs]: You should test the content too

File: integtests/IntegTests/HBVersion.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace IntegTests
+{
+    class HBVersion
```
Comment (bat3) [inline integtests/IntegTests/HBVersion.cs]:
```suggestion
    public class HBVersion
```
Is it working ? I have errors when launching version tests

File: integtests/IntegTests.sln [hunk 4]
```diff
@@ -0,0 +1,25 @@
+﻿
```
Comment (bat3) [inline integtests/IntegTests.sln]: I think you should add the integTest project to the business solution

**Review by bat3 — CHANGES_REQUESTED** [review]

File: integtests/IntegTests.sln (hunk 4 above)
Comment (cquennouelle) [reply integtests/IntegTests.sln]: I'm not sure, if I should test the software as a black box, our tests can be completely decoupled for the actual implementation. Don't you think so?

File: integtests/IntegTests/HBSeriesDto.cs (hunk 1 above)
Comment (cquennouelle) [reply integtests/IntegTests/HBSeriesDto.cs]: IMHO it is not duplication, it is decoupling.

File: integtests/IntegTests/HBVersion.cs (hunk 3 above)
Comment (cquennouelle) [reply integtests/IntegTests/HBVersion.cs]: Not on my machine. I'm on it to see what happens.

File: integtests/IntegTests/SeriesApiTest.cs (hunk 2 above)
Comment (cquennouelle) [reply integtests/IntegTests/SeriesApiTest.cs]: You're right. I'll elaborate this test.

File: integtests/IntegTests.sln (hunk 4 above)
Comment (bat3) [reply integtests/IntegTests.sln]: There is no impact to be in the same solution

File: integtests/IntegTests/HBSeriesDto.cs (hunk 1 above)
Comment (bat3) [reply integtests/IntegTests/HBSeriesDto.cs]: Ok

File: business/integtests/PacsMirrorApiTest.cs [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+            List<HBSeriesDto> seriesList = seriesClient.getAllSeries();
+            Assert.Equal(2, seriesList.Count);
+
+            RestRequest getPacsMirrorRequest = new RestRequest("/api/PacsMirror", DataFormat.Json);
+
+            var responsePost = seriesClient.Client.Get(getPacsMirrorRequest);
+            Assert.Equal(HttpStatusCode.OK, responsePost.StatusCode);
+
+            List<HBSeriesDto> seriesListEmpty = seriesClient.getAllSeries();
+            Assert.Empty(seriesListEmpty);
```
Comment (bat3) [inline business/integtests/PacsMirrorApiTest.cs]: You should just check that TestGetSeries1 and TestGetSeries2 are not present in getAllSeries  
If you have series in your pacs they will be getted

**Review by bat3 — CHANGES_REQUESTED** [review]

**Review by cquennouelle — COMMENTED** [review]
This work is awesome.

**Review by bat3 — APPROVED** [review]

**Pull request merged**